### DISCUSSION TITLE
Disable pdf extension alternatives

### DISCUSSION THREAD

**Discussion by easaint** [cefsharp/CefSharp#4898] [General]
I just see "126.2.7+ disable-pdf-extension no longer supported"

How can I download pdf directly with cefsharp latest version 126.xxx? is there a method to download pdf automatically  without manually clicking the download button ?

it won't go into IDownloadHandler interface when load PDF view. So I have no idea how to save PDF to local with 126.xxx version when finished loading. before 126.x version, I used the param "settings.CefCommandLineArgs.Add("disable-pdf-extension","1");", it can download pdf directly.

**Comment by amaitland** [comment]
Please don't use the feature request template to ask support questions.

https://github.com/cefsharp/CefSharp/issues/4842#issuecomment-2241756143

The `upstream` issue has an alternative option for disabling the pdf viewer. 

If you are unsure of how to set preferences then read the general usage guide.

**Reply by lengkristy** [reply]
OK, thanks
